float64 depth
float64 pitch
